api/server/server.go

```diff
@@ -1062,14 +1062,11 @@ func (s *Server) postContainersWait(version version.Version, w http.ResponseWrit
 		return fmt.Errorf("Missing parameter")
 	}
 
-	name := vars["name"]
-	cont, err := s.daemon.Get(name)
-	if err != nil {
+	status, err := s.daemon.ContainerWait(vars["name"], -1*time.Second)
+	if err != nil && !strings.Contains(err.Error(), "Timed out") {
```

Comment:
This error only happens if container.WaitStop time out. Could make sense to create a new typed error in daemon/container and type checking it here instead of going with strings matching?